### PR TITLE
<UPD>: Split Default Methods files

### PR DESCRIPTION
**Pull request by SEAJang**
# Pull Request Key Information

## Reason for change

Improve user experience by providing a more tailored default method file for each of the stationary and mobile methods.

## What was changed

- simulator expects different inputs based on the method deployment type
- users must now use the correct default parameters for the given deployment types

## Intended Purpose

Default method files have been split into two files, one for stationary and one for mobile methods

## Testing Completed

All unit tests pass
[results.txt](https://github.com/user-attachments/files/15783089/results.txt)

Manually checked to see if the simulation would run using the following parameter files
[simple_test_case1.zip](https://github.com/user-attachments/files/15783092/simple_test_case1.zip)

Additional Info:
 - The sensitivity module was not touched with this change, and may require updates

### CODE REVIEW

File: LDAR_Sim/src/file_processing/input_processing/input_manager.py [hunk 1]
```diff
@@ -273,7 +273,10 @@ def parse_parameters(self, new_parameters_list: list[dict]):
             # methods pre-specified
             for midx, method in program[pc.Levels.METHOD].items():
                 if ParameterProcessingConst.DEFAULT_PARAM_STRING not in method:
-                    def_file = df.METH_DEF_FILE
+                    if method[pc.Method_Params.DEPLOYMENT_TYPE] == pc.Deployment_Types.MOBILE:
+                        def_file = df.METH_MOBILE_DEF_FILE
+                    else:
```
Comment (ThomasGalesloot) [inline LDAR_Sim/src/file_processing/input_processing/input_manager.py:278]: Can we make this an else if and an else with an error for invalid deployment type

File: LDAR_Sim/src/programs/site_level_method.py [hunk 2]
```diff
@@ -88,13 +88,13 @@ def __init__(
             self._inst_threshold = float("inf")
         self._follow_up_schedule: FollowUpMobileSchedule = follow_up_schedule
         self._detection_count = 0
-        self._small_window = properties[pdc.Method_Params.ROLLING_AVRG][
-            pdc.Method_Params.SMALL_WINDOW
-        ]
-        self._large_window = properties[pdc.Method_Params.ROLLING_AVRG][
-            pdc.Method_Params.LARGE_WINDOW
-        ]
         if self._deployment_type == pdc.Deployment_Types.STATIONARY:
```
Comment (ThomasGalesloot) [inline LDAR_Sim/src/programs/site_level_method.py:91]: Can we add a try catch for key error that could occur if one of these are missing and print out an error?

Comment (ThomasGalesloot) [reply LDAR_Sim/src/programs/site_level_method.py:91]: Also for the mobile only parameters can we do something similar?

File: LDAR_Sim/src/programs/method.py [hunk 3]
```diff
@@ -100,12 +99,12 @@ def initialize_crews(self, crews, sites: "list[Site]") -> None:
...
-            self.cost = cost_properties[pdc.Method_Params.PER_SITE]
-        elif cost_properties[pdc.Method_Params.PER_DAY] > 0:
+        if cost_properties[pdc.Method_Params.PER_DAY] > 0:
             self.cost_type = self.PER_DAY_COST
             self.cost = cost_properties[pdc.Method_Params.PER_DAY]
+        elif cost_properties[pdc.Method_Params.PER_SITE] > 0:
+            self.cost_type = self.PER_SITE_COST
+            self.cost = cost_properties[pdc.Method_Params.PER_SITE]
         else:
             self.cost_type = self.PER_SITE_COST
```
Comment (ThomasGalesloot) [inline LDAR_Sim/src/programs/method.py:109]: Should we error out here if we get here with stationary?

**Review by ThomasGalesloot — CHANGES_REQUESTED** [review]
Great work - Just a little of work left on error handling - see my comments

File: LDAR_Sim/src/programs/method.py (hunk 3 above)
Comment (SEAJang) [reply LDAR_Sim/src/programs/method.py:109]: Handled it such that the simulation should still run when no costs are given.

File: LDAR_Sim/src/programs/site_level_method.py (hunk 2 above)
Comment (SEAJang) [reply LDAR_Sim/src/programs/site_level_method.py:91]: That won't ever happen due to the default parameters existing.

**Review by ThomasGalesloot — APPROVED** [review]

**Pull request merged**